modules/Quiz/resources/quiz.js

```diff
@@ -218,6 +219,13 @@
                     };
                 }
             });
+            $( window ).on( "orientationchange", function() {
+                if(_this.ivqShowScreenMode && _this.canShowPortraitWarning() ) {
```

Comment:
canShowPortraitWarning to isPortrait ? 
also, maybe there is an existing check today on the CSS class level? (not sure - asking)